.gitignore

```diff
@@ -2,6 +2,9 @@
 /node_modules
 /bin
 
+# git-crypt
+tegGenKey
```

Comment:
I see. But I think it's still safer to ignore it just in case someone like me carelessly put it in the project directory